step.yml

```diff
@@ -0,0 +1,36 @@
+title: Disable iphonesimulator ConnectHardwareKeyboard preference
+summary: |
+  This Step disables ConnectHardwareKeyboard preference for all iphonesimulator devices.
+website: https://github.com/bitrise-steplib/steps-disable-iphonesimulator-connect-hardware-keyboard
+source_code_url: https://github.com/bitrise-steplib/steps-disable-iphonesimulator-connect-hardware-keyboard
+support_url: https://github.com/bitrise-steplib/steps-disable-iphonesimulator-connect-hardware-keyboard/issues
+
+project_type_tags:
+- ios
+- react-native
+- cordova
+- ionic
+- flutter
+
+type_tags:
+- utility
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-disable-iphonesimulator-connect-hardware-keyboard
+
+inputs:
+- iphonesimulator_preferences_pth: ~/Library/Preferences/com.apple.iphonesimulator.plist
```

Comment:
This is a value that the user should not care about normally. What if we add the `is_dont_change_value` to it so they cannot change it in the WFE UI (just in the yaml if it will be ever needed)?

Reply:
Updated here: https://github.com/bitrise-steplib/steps-disable-iphonesimulator-connect-hardware-keyboard/pull/1/commits/958d48ea3532bb884a762c17a8c5fe70f93495ee